### PR TITLE
whisper is no longer a part of parity-ethereum repo

### PR DESCRIPTION
**Pull request by debris**
it's moved over here: https://github.com/paritytech/whisper

* several README files still need to be updated

### CODE REVIEW

File: parity/cli/mod.rs [hunk 1]
```diff
@@ -926,11 +926,11 @@ usage! {
 		["Whisper Options"]
 			FLAG flag_whisper: (bool) = false, or |c: &Config| c.whisper.as_ref()?.enabled,
 			"--whisper",
-			"Enable the Whisper network.",
+			"Does nothing. Whisper is no longer a part of parity-ethereum cli.",
```
Comment (dvdplm) [inline parity/cli/mod.rs]: Link to new repo?

**Comment by dvdplm** [comment]
cc @ltfschoen you have a few pending PRs in this area. Do you think you can move them over to the new repo and also take a look at the Readme(s)/docs/wiki?

**Comment by ltfschoen** [comment]
> cc @ltfschoen you have a few pending PRs in this area. Do you think you can move them over to the new repo and also take a look at the Readme(s)/docs/wiki?

Yes, but I'll wait for this to get finalised and merged first, then I'll makes those changes in separate PRs

File: parity/cli/mod.rs (hunk 1 above)
Comment (debris) [reply parity/cli/mod.rs]: fixed!

**Comment by debris** [comment]
@ordian please review :)

File: scripts/gitlab/build-linux.sh [hunk 2]
```diff
@@ -27,7 +27,6 @@ else
   time cargo build --target $CARGO_TARGET --verbose --color=always --release -p evmbin
   time cargo build --target $CARGO_TARGET --verbose --color=always --release -p ethstore-cli
   time cargo build --target $CARGO_TARGET --verbose --color=always --release -p ethkey-cli
-  time cargo build --target $CARGO_TARGET --verbose --color=always --release -p whisper-cli
```
Comment (ordian) [inline scripts/gitlab/build-linux.sh]: Does that mean we no longer provide whisper binaries?

File: parity/run.rs [hunk 3]
```diff
@@ -286,7 +276,7 @@ fn execute_light_impl<Cr>(cmd: RunCmd, logger: Arc<RotatingLogger>, on_client_rq
 		network_id: cmd.network_id.unwrap_or(spec.network_id()),
 		subprotocol_name: sync::LIGHT_PROTOCOL,
 		handlers: vec![on_demand.clone()],
-		attached_protos: attached_protos,
+		attached_protos: Vec::new(),
```
Comment (ordian) [inline parity/run.rs]: is it used at all?

File: scripts/gitlab/build-linux.sh (hunk 2 above)
Comment (dvdplm) [reply scripts/gitlab/build-linux.sh]: Yes.

cc @s3krit

File: parity/run.rs (hunk 3 above)
Comment (dvdplm) [reply parity/run.rs]: Doesn't seem so. Good catch. @debris remove it?

**Comment by dvdplm** [comment]
> separate PR?

I'd prefer it happened here tbh. It can be useful to have the extraction all in one commit for posterity.

**Comment by dvdplm** [comment]
@debris there's also the config generator to cleanup: https://github.com/paritytech/parity-config-generator/issues/50

File: parity/run.rs (hunk 3 above)
Comment (debris) [reply parity/run.rs]: done

File: parity/deprecated.rs [hunk 4]
```diff
@@ -229,6 +229,16 @@ pub fn find_deprecated(args: &Args) -> Vec<Deprecated> {
 		result.push(Deprecated::Removed("--ntp-servers"));
 	}
 
+	// Removed in 2.6.
```
Comment (ordian) [inline parity/deprecated.rs]:
```suggestion
	// Removed in 2.7.
```

**Pull request merged**